src/components/Button/src/Button.stories.tsx

```diff
@@ -1,20 +1,20 @@
-import React from 'react';
-// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
-import { Meta, Story } from '@storybook/react/types-6-0';
-import Button, { ButtonProps } from '.';
-import pkg from '../package.json';
+import React from "react";
```

Comment:
Check your prettier plugin configuration, this should be single quotes

Reply:
Thanks! I fixed my config but didn't rerun it for that file